frontend/packages/console-app/src/components/cloud-shell/__tests__/CloudShellTerminal.spec.tsx

```diff
@@ -19,6 +19,14 @@ jest.mock('react-i18next', () => {
   };
 });
 
+jest.mock('@console/shared', () => {
+  const ActualShared = require.requireActual('@console/shared');
+  return {
+    ...ActualShared,
+    useUserSettingsCompatibility: () => ['', () => {}],
+  };
+});
```

Comment:
I don't think we need requireActual here.
```suggestion
jest.mock('@console/shared/src/hooks/useUserSettingsCompatibility', () => {
  return {
    useUserSettingsCompatibility: () => ['', () => {}],
  };
});
```